src/eval_framework/context/determined.py

```diff
@@ -48,6 +48,7 @@ class Hyperparameters(BaseModel):
     output_dir: Path
     hf_upload_dir: str | None = None
     hf_upload_repo: str | None = None
+    wandb_project: str | None = None
```

Comment:
Is it realistic that `wandb_project` would differ across eval runs in a single determined config?

Reply:
I just thought this is something no probably being varied and these things are then passed via CLI (e.g. `--save-logs`) but since eg `hf_upload_repo` is here then let's keep it here as well.